h/schemas/api/user.py

```diff
@@ -55,13 +56,28 @@ class CreateUserAPISchema(JSONSchema):
                 }
             },
         },
-        'required': [
-            'authority',
-            'username',
-            'email',
-        ],
+        'anyOf': [  # email may be empty if identities are present
+            {
+                'required': [
+                    'authority',
+                    'username',
+                    'email',
+                ],
+            },
+            {
+                'required': [
+                    'authority',
+                    'username',
+                    'identities',
+                ],
+            }
+        ]
     }
 
+    def validate(self, data):
+        appstruct = super(CreateUserAPISchema, self).validate(data)
+        return appstruct
```

Comment:
Wow! Excellent catch. I had put this here because I thought I was going to need extra validation logic before I realized JSONSchema alone would do the trick. Will remove.